packages/webapp/src/_app/layouts/card.tsx

```diff
@@ -0,0 +1,18 @@
+import { Heading } from "../ui";
+
+interface Props {
+    title?: string;
+    titleSize?: 1 | 2 | 3 | 4;
+    children: React.ReactNode;
+}
+
+export const Card = ({ children, title, titleSize = 1 }: Props) => (
```

Comment:
When I think of Cards, I usually think about a component.... This `layouts` folder I created thinking about "page-layouts". I'd probably rename this Card to another "containerish/layouty" kinda name because I believe at some point we will want to create a generic card to use for posts, feeds etc. Unless you think this is the card component for it, so I'd probably move it to the ui components folder.

Reply:
Yeah, I struggled a bit with where to put this one. I tried to keep the component super simple, because I do envision us eventually using this elsewhere, and multiple times per page. For example, on the members list page, it's used twice. I'll move it to `ui` for now. Thanks!